### ISSUE TITLE
leverage modified h,j,k,l keys

### ISSUE THREAD

**Issue by Parasrah** [Parasrah/kak-config#23]
blocked by ncurses for now, but modern terminals have support for the modifiers (control/alt) being used with `hjkl`. Would be nice to have jump commands mapped using this

**Comment by Parasrah** [comment]
tried this recently, still doesn't work with kitty
